applications/README.md

```diff
@@ -1,7 +1,87 @@
 ```text
 SPDX-License-Identifier: Apache-2.0
-Copyright (c) 2020 Intel Corporation
+Copyright (c) 2020 Actian Corporation
 ```
 
-## OpenNESS Applications
-This folder contains applications that are supported by OpenNESS Network and On-Premises Edge. Applications include that services end user and also those that provide services to other applications in the edge cloud.
+# **Qwilt CDN Overview**
```

Comment:
This top-level README is for the whole edgeapps repo. For the Qwilt-based CDN app, please create the readme file at path `applications/qwilt/README.md` in pull-request #30.